test/integration/framework/src/siftool/test_utils.py

```diff
@@ -575,7 +609,13 @@ def wait_for_sif_balance_change(self, sif_addr, old_balances, min_changes: Itera
             new_balances = self.get_sifchain_balance(sif_addr)
             delta = sifchain.balance_delta(old_balances, new_balances)
             if min_changes is not None:
-                if all(denom in delta for (_, denom) in min_changes):
+                reached_changes = 0
```

Comment:
While you did not create this function you are modifying it, please add type hints to the function params and return type.